src/LabelSelector/LabelChip/LabelChip.tsx

```diff
@@ -16,6 +16,7 @@ export default function LabelChip({
   return (
     <Chip
       {...props}
+      className="label-chip"
```

Comment:
Not directly using this anywhere yet but also made sense to add class for the base label chip component as I was here.